tests/test_permutation.py

```diff
@@ -1,47 +1,82 @@
 # -*- coding: utf-8 -*-
 """
-Testing File
+Testing the functions in permutation.py
 """
-# Import necessary modules
 import numpy as np
 import pyrft as pr
-import os
-
-# The location where the pyrft module is saved (need to specify yourself)
-pyrft_loc = 'C:/Users/12SDa/davenpor/davenpor/Toolboxes/pyrft/'
-
-# Where data is stored
-data_store = pyrft_loc + 'tests2/test_data/'
 
 def test_boot_contrasts():
+    """ Testing the boot_contrasts function """
     np.random.seed(10)
-    Dim = 5; N = 30; categ = np.random.multinomial(2, [1/3,1/3,1/3], size = N)[:,1]
-    X = pr.groupX(categ); C = np.array([[1,-1,0],[0,1,-1]]); lat_data = pr.wfield(Dim,N)
-    minP, orig_pvalues, pivotal_stats, _ = pr.boot_contrasts(lat_data, X, C)
-    
-    file_name = 'test_boot_contrasts_data.npz'
-    file_loc = data_store + file_name
-    if not os.path.exists(file_loc):
-       np.savez(file_loc, minP = minP, orig_pvalues = orig_pvalues.field, pivotal_stats = pivotal_stats)
-    else:
-       results = np.load(file_loc, allow_pickle = True)
-       assert((minP == results['minP']).all())
-       assert((orig_pvalues.field == results['orig_pvalues']).all())
-       assert((pivotal_stats == results['pivotal_stats']).all())
+    for i in np.arange(2):
+        if i == 1:
+            # 1D example
+            dim = 5
+        else:
+            # 2D example
+            dim = (10,10)
+

```

Comment:
the above loop simply creates dim=(10, 10)

Reply:
oh yeah, thanks. Fixed now!